library/route.h

```diff
@@ -46,14 +46,14 @@ class dnet_route_list
 
 extern "C" {
 #else // __cplusplus
-typedef struct dnet_route_list_t dnet_route_list;
+typedef struct dnet_route_list dnet_route_list;
```

Comment:
It is used as forward declaration for undefined structure. It could be replaced simpler with classic forward declaration.